src/gurobi_logtools/parsers/continuous.py

```diff
@@ -103,9 +134,16 @@ def parse(self, line: str) -> ParseResult:
     def get_summary(self) -> Dict:
         """Return the current parsed summary."""
         self._summary.update(self._barrier_parser.get_summary())
+        self._summary.update(self._pdhg_parser.get_summary())
+        self._summary.update(self._crossover_parser.get_summary())
         self._summary.update(self._simplex_parser.get_summary())
         return self._summary
 
     def get_progress(self) -> list:
         """Return the detailed progress in the continuous method."""
-        return self._barrier_parser.get_progress() + self._simplex_parser.get_progress()
+        return (
+            self._barrier_parser.get_progress()
+            + self._pdhg_parser.get_progress()
+            + self._crossover_parser.get_progress()
+            + self._simplex_parser.get_progress()
+        )
```

Comment:
Run time isn't super critical, and I'm being pedantic but can we use this pattern?

```python
        return [
            *self._barrier_parser.get_progress(),
            *self._pdhg_parser.get_progress(),
            *self._crossover_parser.get_progress(),
            *self._simplex_parser.get_progress(),
        ]
```